c++/src/kj/async-test.c++

```diff
@@ -509,6 +509,47 @@ TEST(Async, Fork) {
   EXPECT_EQ(789, branch2.wait(waitScope));
 }
 
+TEST(Async, ForkMaybeOwn) {
+  struct MyType {
+    int value = 123;
+
+    Own<MyType> addRef() const {
+      return kj::heap<MyType>(MyType{ .value = 456, });
+    }
+  };
+  struct MyType2 {
```

Comment:
Unused type?

FWIW the next test below tests the addRef() behavior of forked promises and defines `RefcountedInt` for that purpose. Might have been easier to copy/paste that test as a starting point.

Reply:
Yeah, much better suggestion. Thanks!